.github/workflows/PR-into-2021-1.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           
           # actual merge operation
           git merge --no-ff origin/mps/2020.3 --
-          git checkout --ours -- conflictTest.file
+          git checkout --ours -- gradle/libs.versions.toml
```

Comment:
Is that valid. That means, whenever we update an upstream dependency in the original PR, it might not get propagated here because we overwrite the file with the version of the target branch, right?